webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -24,3 +28,39 @@ export const addSearchToURL = (path, query) => {
   url.addSearch({ ...query, include_permissions: true });
   return url.toString();
 };
+
+export const getDuration = (start, finish) => {
+  if (!start && !finish)
+    return { text: __('N/A'), tooltip: __('No start or end dates') };
+
+  if (!start && finish) {
+    return { text: __('N/A'), tooltip: __('Task was canceled') };
+  }
+
+  const dateOptions = {
+    largest: 1,
+    language: documentLocale(),
+    fallbacks: ['en'],
+    round: true,
+  };
+
+  start = new Date(isoCompatibleDate(start));
```

Comment:
Not sure if it is a good idea to amend args,
please declare a new const for that.